### DISCUSSION TITLE
Continuation of [Remix] Error: ENOENT: no such file or directory, copyfile '/var/task/node_modules/@vercel/remix/default-server.js'

### DISCUSSION THREAD

**Discussion by bradymwilliams** [vercel/community#1405] [Help] [answered]
### Summary

Potentially Related to [this discussion](https://github.com/orgs/vercel/discussions/1282).

After the fix for this was deployed this error went away on simple remix sites deploying.

However, in the event of a remix site within a turborepo when the cache results in FULL TURBO this error returns.

![image](https://user-images.githubusercontent.com/7980606/215868376-f36e8ada-2e3c-47b2-90db-77a1ebd2070f.png)

This build failure results in failing checks on pull requests.

### Additional information

```


site:build: Building Remix app in production mode...
--
12:57:17.839 | wisecapital:build: Built in 338ms
12:57:17.839 |  
12:57:17.839 | Tasks:    1 successful, 1 total
12:57:17.839 | Cached:    1 cached, 1 total
12:57:17.839 | Time:    781ms >>> FULL TURBO
12:57:17.839 |  
12:57:17.915 | Error: ENOENT: no such file or directory, copyfile '/var/task/node_modules/@vercel/remix/default-server.js' -> '/vercel/path0/apps/site/build/__vc_handler.js'


```

**Comment by swarnava** [comment, marked as answer]
Hi,

It looks like you already have a case with us (00119637). We will communicate via same thread. Thank you!
